### DISCUSSION TITLE
I try to style my H1 but it doesn't work

### DISCUSSION THREAD

**Discussion by Nassir-Al** [neuefische/questions#248] [Web] [answered]
Hier ist my PR:
https://github.com/Nassir-Al/capstone-project/pull/12

i try to style it like this but it didn't work:
<img width="520" alt="Bildschirmfoto 2022-10-12 um 11 49 25" src="https://user-images.githubusercontent.com/109968270/195310812-bee00e49-75cd-45ec-b36f-dd3ee8d6efa3.png">

this is my result:
<img width="551" alt="Bildschirmfoto 2022-10-12 um 11 36 14" src="https://user-images.githubusercontent.com/109968270/195311616-adca8acf-d773-4c22-8e71-09fd0c20cdf3.png">

**Comment by mbosselmann** [comment, marked as answer]
Hi Nassir,
your component `Div_Headline1` needs to be fixed:

The current state:
```css
export const Div_Headline1 = styled.div`
	border: 1px;
	border-radius: 1px solid black;
`;
```

You have confused two declarations with each other: `border` and `border-radius`. `1px solid black` should be the value for the property `border` and `1px` should be the value for the property `border-radius`. :)

**Reply by mbosselmann** [reply]
I forgot to mention that you don't need an extra `div` element, you can apply the declaration to your `h1` element directly. Also the naming is not good. Don't use numbers, use meaningful names.

**Comment by Nassir-Al** [comment]
Ok, thanks for your answer!
